fMRIVolume/scripts/OneStepResampling.sh

```diff
@@ -180,9 +217,13 @@ for ((k=0; k < $NumFrames; k++)); do
   fi
   prevmatrix="${MotionMatrixFolder}/${MotionMatrixPrefix}${vnum}"
 
-  # Combine GCD with motion correction
-  ${FSLDIR}/bin/convertwarp --relout --rel --ref=${WD}/prevols/vol${vnum}.nii.gz --warp1=${GradientDistortionField} --postmat=${MotionMatrixFolder}/${MotionMatrixPrefix}${vnum} --out=${MotionMatrixFolder}/${MotionMatrixPrefix}${vnum}_gdc_warp.nii.gz
-  # Add in the warp to MNI152
+  ###Add stuff for RMS###
```

Comment:
The comment here `###Add stuff for RMS###` should be eliminated, and the comments that I had added
`# Combine GCD with motion correction`
and
`# Add in the warp to MNI152`
restored to the appropriate locations.

Reply:
Fixed with pulling in the latest changes.